tcms/testplans/views.py

```diff
@@ -1005,7 +1005,7 @@ def link_cases(self, template_name='plan/search_case.html'):
                                       context_instance=RequestContext(request))
 
         def delete_cases(self):
-            if not request.GET.get('case'):
+            if not request.POST.get('case'):
```

Comment:
Right. To fix this, need to split ``CaseActions`` into separate view methods, for example,

```
@require_POST
def delete_cases(request, ...):
    ...

@require_POST
def order_cases(request, ...):
    ...
```

This could be fixed in another PR, but not this one.